lisa/tools/lagscope.py

```diff
@@ -232,10 +232,10 @@ def run_as_client(
 
     def get_average(self, result: ExecutableResult) -> Decimal:
         matched_results = self._average_pattern.match(result.stdout)
-        assert (
-            matched_results
-        ), "not found matched average latency statistics from lagscope results."
-        return Decimal(matched_results.group("average_latency_us"))
+        if matched_results:
+            return Decimal(matched_results.group("average_latency_us"))
+        else:
+            return Decimal(-1.0)
```

Comment:
Add debug log to explain why the average_latency_us is not matched, and -1 returned.